src/web/components/AdSlot.tsx

```diff
@@ -9,7 +9,6 @@ import { from } from '@guardian/src-foundations/mq';
 export const labelStyles = css`
     .ad-slot__label {
         ${textSans.xsmall()};
-        position: relative;
```

Comment:
`labelStyles` is also used in `MostViewedFooter` and `ArticleContainer`. I can't see an impact to removing it but just flagging it up so we're sure.

Reply:
Yup, I couldn't see an absolutely positioned content within so think it might have been a copy paste legacy...